pkg/controller/namespace/namespace_controller.go

```diff
@@ -468,13 +477,13 @@ func deletePersistentVolumeClaims(kubeClient clientset.Interface, ns string) err
 	return nil
 }
 
-func deleteHorizontalPodAutoscalers(expClient extensions_unversioned.ExtensionsInterface, ns string) error {
```

Comment:
Not changing as explained in https://github.com/kubernetes/kubernetes/pull/20501#issuecomment-180260757